cloud/api-gateway/src/main/java/com/qpay/apigateway/config/WebClientConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.qpay.apigateway.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class WebClientConfig {
+
+    @Bean
+    public WebClient.Builder webClientBuilder() {
+        return WebClient.builder();
+    }
+
+    @Bean
+    public WebClient webClient(final WebClient.Builder webClientBuilder) {
+        return webClientBuilder.build();
+    }
```

Comment:
it's really strange to split it into 2 beans. Any reason for it?

Reply:
thought it would be good to separate if i want to create webclient with different implementations, but i see it's stupid. i will remove